pages/dkp/konvoy/2.1/choose-infrastructure/verify-installation/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle: 
+title: Verify Konvoy installation
+menuWeight: 50
+excerpt: Check DKP components to verify the status of your cluster
+enterprise: false
+beta: false
+---
+
+In this section, we show you how to verify a Konvoy installation.
```

Comment:
```suggestion
In this section we show you how to verify a Konvoy installation.
```

Reply:
I suggest getting rid of the meta reference:

```suggestion
How to verify a Konvoy installation.
```
 
 ... but this still leaves me with the question, "verify it what?"

Reply:
Also, is it still "konvoy"?

Reply:
It is still Konvoy because this is in the Konvoy docs - I think this may have to change down the road, but not in this PR. Good callout, though.